src/components/SearchBarInput.jsx

```diff
@@ -17,4 +17,8 @@ function SearchBarInput() {
   )
 }
 
-export default SearchBarInput
\ No newline at end of file
+<<<<<<< HEAD
+export default SearchBarInput;
+=======
+export default SearchBarInput
```

Comment:
this is a component that i had built, did you change something that i am not seeing or what happened can you explain a bit?